### ISSUE TITLE
humanOutputGenerator is not a function

### ISSUE THREAD

**Issue by jonathanong** [Azure/azure-xplat-cli#3302]
CLI Version: **0.10.7**
OS Type: Mac
Installation via:  npm

Mode: **ARM / ASM**

Description:
`webapp config appsettings set` throws error

Error stack trace:

```
2016-11-07T08:23:43.172Z:
{ TypeError: humanOutputGenerator is not a function
  <<< async stack >>>
    at Interactor.formatOutput (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/interaction.js:307:7)
    at __2 (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/commands/arm/webapp/webapp.config.js:255:35)
  at __2 (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/commands/arm/webapp/webapp.config.js:252:35)
  <<< raw stack >>>
    at Interactor.formatOutput (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/interaction.js:307:7)
    at __$__2 (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/commands/arm/webapp/webapp.config.js:255:35)
    at ___ (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/commands/arm/webapp/webapp.config.js:253:94)
    at ___ (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:111:13)
    at /Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/node_modules/azure-arm-website/lib/operations/sites.js:13210:12
    at retryCallback (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/node_modules/ms-rest/lib/filters/exponentialRetryPolicyFilter.js:140:9)
    at handleRedirect (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/node_modules/ms-rest/lib/filters/redirectFilter.js:36:9)
    at /Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/utils.js:587:7
    at handleRedirect (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/utils.js:564:9)
    at /Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/logging.js:339:7
  stack: [Getter/Setter],
  __frame: 
   { name: '__2',
     line: 165,
     file: '/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/commands/arm/webapp/webapp.config.js',
     prev: undefined,
     calls: 0,
     active: false,
     offset: 87,
     col: 34 },
  rawStack: [Getter] }
TypeError: humanOutputGenerator is not a function
  <<< async stack >>>
    at Interactor.formatOutput (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/interaction.js:307:7)
    at __2 (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/commands/arm/webapp/webapp.config.js:255:35)
  at __2 (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/commands/arm/webapp/webapp.config.js:252:35)
  <<< raw stack >>>
    at Interactor.formatOutput (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/interaction.js:307:7)
    at __$__2 (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/commands/arm/webapp/webapp.config.js:255:35)
    at ___ (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/commands/arm/webapp/webapp.config.js:253:94)
    at ___ (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:111:13)
    at /Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/node_modules/azure-arm-website/lib/operations/sites.js:13210:12
    at retryCallback (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/node_modules/ms-rest/lib/filters/exponentialRetryPolicyFilter.js:140:9)
    at handleRedirect (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/node_modules/ms-rest/lib/filters/redirectFilter.js:36:9)
    at /Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/utils.js:587:7
    at handleRedirect (/Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/utils.js:564:9)
    at /Users/jong/.nvm/versions/node/v6.9.1/lib/node_modules/azure-cli/lib/util/logging.js:339:7
```

```
$ azure --version
0.10.7 (node: 6.9.1)
```

**Comment by jonathanong** [comment]
downgrading to `0.10.6` worked. no idea why `0.10.7` is published when i only see `0.10.6` in this repo

**Comment by balajikris** [comment]
`0.10.7` is indeed the latest release. we'll investigate to reproduce the issue on `0.10.7`, thanks for reporting this.
